docs/project/contribute.md

```diff
@@ -64,13 +64,39 @@ Search is by algolia, the configuration is at <https://github.com/algolia/docsea
 
 ## Running the site locally:
 
-All Contributors uses Docusaurus to maintain our documentation website. For more info visit the [Docusaurus Website](https://docusaurus.io)
+All Contributors currently uses [Docusaurus 1.x](https://docusaurus.io) for our documentation website.
 
-To get started locally:
+To build the docs locally:
 
-- `yarn install`
-- `yarn start`
-- go to `http://localhost:3000` (Refresh the page for your changes to be reflected)
+* https://tutorial.docusaurus.io/docs/intro
+
+1. Make sure `npm` is installed on your machine. Use the [nodejs documentation](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) if you aren't sure how to install `npm`. 
+
+To check that npm is installed run the following in your favorite shell:
+
+`npm -v`
+
+Next, make sure you have atleast node version 0.18 or higher. To check the version,
+use:
+
+`node -v`
+
+If you don't have atleast version `.18` or higher, please install node or upgrade your current version.
+
+Install the newest (deprecated) version of Docusaurus:
```

Comment:
Do they not have an undeprecated version?

Reply:
They do. Can you [please have a look as this open issue](https://github.com/all-contributors/all-contributors/issues/843) . Essentially if we want to move the entire site to the newest version, we will need to update the structure of the documentation. This structure will impact crowdin's setup. We are 2 versions behind on these docs. SO I think because I don't have access to crowdin beyond adding translations I don't want to break translations until we know we have full admin access to platforms (or we have a plan in place to migrate translations.

Reply:
Update - thank you for giving me access to crowdin! we can now figure out how to sync github pages with translation content and also setup github pages 🚀 I suggest still that we get the site live as is FIRST and then we migrate to Docusaurus 3.x and update all deps at that time!!

Reply:
Yeah, getting it working then migrating to a later version of Docusaurus is a good plan.

Reply:
Ok perfect. I also saw the updates you made to the pr title. I'll read a bit more about angular / conventional commits as well.